bin/ch/Helpers.cpp

```diff
@@ -4,6 +4,212 @@
 //-------------------------------------------------------------------------------------------------------
 #include "stdafx.h"
 
+//TODO: x-plat definitions
+#ifdef _WIN32
+typedef char16 TTDHostCharType;
```

Comment:
Wonder if you could consider using "char16" directly. It is used through out the whole code base. Using it for TTD may be more readable.